app/routes/logout.tsx

```diff
@@ -1,17 +1,16 @@
 import { redirect } from '@remix-run/node';
-import { getUsername } from '~/session.server';
+import { getUsername, logout } from '~/session.server';
 
 import type { ActionArgs } from '@remix-run/node';
-import { createLogoutRequest } from '~/lib/saml.server';
 
 export const action = async ({ request }: ActionArgs) => {
   const user = await getUsername(request);
 
-  // Do the opposite of login
+  // Invalidate the Starchart session but do not log out from Seneca IDP.
   if (user) {
-    const context = createLogoutRequest(user);
-    return redirect(context);
+    return await logout(request);
```

Comment:
I learned this only weeks ago in code review. `await` is redundant when returning a promise.

Reply:
seems like the only viable time to use [await](https://eslint.org/docs/latest/rules/no-return-await) is when you're trying to catch and error from another function

Reply:
> seems like the only viable time to use [await](https://eslint.org/docs/latest/rules/no-return-await) is when you're trying to catch and error from another function

I don't think this is true

Reply:
However I do think Won might be right about it being redundant here.